test/ConfigStoreClient.ts

```diff
@@ -367,6 +367,43 @@ describe("AcrossConfigStoreClient", function () {
         configStoreClient.getMaxL1TokenCountForPoolRebalanceLeafForBlock(initialUpdate.blockNumber - 1)
       ).to.throw(/Could not find MaxL1TokenCount/);
     });
+    it("Should test lite chain ID updates", async function () {
+      const update1 = await configStore.updateGlobalConfig(
+        utf8ToHex(GLOBAL_CONFIG_STORE_KEYS.LITE_CHAIN_ID_INDICES),
+        JSON.stringify([1])
+      );
+      let timestamp = (await configStore.provider.getBlock(update1.blockNumber!)).timestamp;
+      expect(timestamp).to.not.be.undefined;
+
+      // Set the bounds for the new lite chain
+      const timestampBeforeNewLiteChains = timestamp - 5;
+      const timestampAfterNewLiteChains = timestamp + 5;
+
+      const blockBeforeNewLiteChains = update1.blockNumber! - 1;
+      const blockAfterNewLiteChains = update1.blockNumber! + 1;
+
+      await configStoreClient.update();
+      // Test the getliteChainIdIndicesForTimestamp function
+      expect(configStoreClient.getliteChainIdIndicesForTimestamp(timestampBeforeNewLiteChains)).to.deep.equal([]);
+      expect(configStoreClient.getliteChainIdIndicesForTimestamp(timestampAfterNewLiteChains)).to.deep.equal([1]);
+      // Test the getliteChainIdIndicesForBlock function
+      expect(configStoreClient.getLiteChainIdIndicesForBlock(blockBeforeNewLiteChains)).to.deep.equal([]);
+      expect(configStoreClient.getLiteChainIdIndicesForBlock(blockAfterNewLiteChains)).to.deep.equal([1]);
+
+      const update2 = await configStore.updateGlobalConfig(
+        utf8ToHex(GLOBAL_CONFIG_STORE_KEYS.LITE_CHAIN_ID_INDICES),
+        JSON.stringify([1, 15])
```

Comment:
What if there is an invalid update? i.e. what if there is a chain ID that isn't valid? or not a number?

Reply:
Resolved below